app/views/expenses/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= simple_form_for expense do |f| %>
+  <%= f.input :name, placeholder: "Your name" %>
+  <%= f.input :due_date, order: [:day, :month, :year] %>
+  <%= f.input :frequency, placeholder: "weekly, fortnightly, monthly" %>
+  <%= f.input :bill_estimate, placeholder: "how much you think", input_html: { value: number_with_precision(f.object.bill_estimate, precision: 2) } %>
```

Comment:
Use translations

Reply:
Also length of line

Reply:
Duh, working too late for my brain again! Done in sha: 740613b